lib/lita-lean_tc.rb

```diff
@@ -34,11 +36,13 @@ class Lean < Handler
       route(/\Alean set-types ([a-zA-Z0-9]+)\Z/i, :set_types, command: true, help: { "lean set-types [board id]" => "Begin looping through cards without a type on the nominated trello board"})
       route(/\Alean set-streams ([a-zA-Z0-9]+)\Z/i, :set_streams, command: true, help: { "lean set-streams [board id]" => "Begin looping through cards without a stream on the nominated trello board"})
       route(/\Alean confirmed-cards\Z/i, :list_cards, command: true, help: { "lean confirmed-cards" => "List all cards in the confirmed column" })
+      route(/\Alean list-feature-requests\Z/i, :list_feature_request, command: true, help: { "lean feature-request" => "List all cards on the Feature Request board" })
```

Comment:
The help text is incorrect for this command, it should read "lean list-feature-requests".